examples/cli/README.md

```diff
@@ -40,7 +40,7 @@ Use the CLI to send a [double ratchet message](https://github.com/xmtp/libxmtp/b
 6. Add user 2 to the group
 
    ```bash
-   ./xli.sh --db user1.db3 add-group-member $GROUP_ID $USER_2_ACCOUNT_ADDRESS
```

Comment:
How will multiple account addresses be distinguished from the group ID here? Might want to clarify

Reply:
Good eye. I forgot to hit save on this file, the order is actually backwards. The $GROUP_ID belongs first